terraform/database.tf

```diff
@@ -0,0 +1,192 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+resource "random_string" "db-name" {
+  length  = 5
+  special = false
+  number  = false
+  upper   = false
+}
+
+resource "google_sql_database_instance" "db-inst" {
+  project          = data.google_project.project.project_id
+  region           = var.region
+  database_version = "POSTGRES_11"
+  name             = "en-${random_string.db-name.result}"
```

Comment:
Note: this is a change. It was previously named "contact-tracing-...". @mikehelmick this will trigger a conflict so we'll need to manually rename the server in prod before running Terraform.